packages/toolkit/src/view/catalog/components/CreateCatalogCard.tsx

```diff
@@ -167,7 +217,7 @@ Tokens: #: ${catalog.totalTokens || "N/A"}
         initialValues={{
           name: catalog.name,
           description: catalog.description,
-          tags: catalog.tags || [],
+          // tags: catalog.tags ? catalog.tags : [],
```

Comment:
@thewbuk If this won't be used, let remove it

Reply:
Tags will be used in a near future

Reply:
ok